src/Arcus.Security.Tests.Unit/KeyVault/KeyVaultSecretProviderTests.cs

```diff
@@ -75,7 +75,7 @@ public async Task KeyVaultSecretProvider_GetsSecretValue_AfterRetriedTooManyRequ
             KeyVaultSecretProvider provider = CreateSecretProviderWithTooManyRequestSimulation(expected);
```

Comment:
Probably some duplication of tests here since the obsolete methods are removed.

Reply:
done